Model/Config.php

```diff
@@ -540,14 +542,14 @@ public function getDefaultStoreId(int $websiteId): int
 
     /**
      * Check if Yotpo is enabled and if order sync is active.
-     *
+     * @param int|null $storeId
      * @return bool
      * @throws LocalizedException
      * @throws NoSuchEntityException
      */
-    public function isCatalogSyncActive()
+    public function isCatalogSyncActive($storeId = null)
     {
-        return ($this->isEnabled() && $this->getConfig('catalog_sync_enable'));
+        return ($this->isEnabled() && $this->getConfig('catalog_sync_enable', $storeId));
```

Comment:
What happens if you pass null?
This is a super fundemental method so wanna make sure we're good.